test/mocks/providedDependency.mock.ts

```diff
@@ -1,6 +1,7 @@
 import { SinonStub, stub } from 'sinon';
 
-export const ProvidedDependencyToken = Symbol('providedDependency');
+// Would use Symbol() in ES6, string in ES5
```

Comment:
Same again, the tests should be run on node so you shouldn't have to change this, I'd be happy for you to add a separate test for using strings but the primary use case is ES6 with node.